frontend/pulse3d/src/pages/upload-form.js

```diff
@@ -176,7 +176,9 @@ export default function UploadForm() {
         tabSelection === "Re-analyze Existing Upload" &&
         usageQuota &&
         usageQuota.limits &&
```

Comment:
last nit picky comment, can we break up this big conditional into smaller ones with variable names that explain what it's checking? It's kind of hard to follow. Or at min, add some comments. How does this check if it's not the first reanalysis?